mofo-shareprogress-page-wrapper.css

```diff
@@ -0,0 +1,270 @@
+/* All the !important rules are
+   To override CSS rules injected by Shareprogress' script */
+
+body {
+  background: #b7b9fa; /* Foudation Brand $light-blue */
+  font-family: 'Nunito Sans', sans-serif !important;
```

Comment:
we do not have a google font `<link>` for Nunito

Reply:
We do. In `default-wrapper.html` line 13.
```html
<link href="https://fonts.googleapis.com/css2?family=Nunito+Sans&family=Zilla+Slab:wght@400;500&display=swap" rel="stylesheet">
```

Though I need to double-check if I include the right font weights 😂

Reply:
I am ~ 👀~